cmd/edit/kubeletconfig/cmd.go

```diff
@@ -97,19 +97,15 @@ func run(_ *cobra.Command, _ []string) {
 		r.Reporter.Infof("Enabling interactive mode")
 	}
 
+	maxPidsLimit, err := GetMaxPidsLimit(r.OCMClient)
+	if err != nil {
+		r.Reporter.Errorf("Failed to check maximum allowed Pids limit for for cluster '%s'",
+			clusterKey)
+		os.Exit(1)
+	}
+
 	if interactive.Enabled() {
-		args.podPidsLimit, err = interactive.GetInt(interactive.Input{
-			Question: InteractivePodPidsLimitPrompt,
-			Help:     InteractivePodPidsLimitHelp,
-			Options:  nil,
-			Default:  kubeletconfig.PodPidsLimit(),
-			Required: true,
-			Validators: []interactive.Validator{
-				// We only validate the minimum as some customers have capability to exceed maximum
-				// We rely on backend validation for that
-				interactive.Min(MinPodPidsLimit),
-			},
-		})
+		args.podPidsLimit, err = interactive.GetInt(GetInterativeInput(maxPidsLimit, kubeletconfig))
```

Comment:
GetIntera`c`tiveInput

Reply:
Eyes of a 🥷  Will get it fixed.

cheers,
Rob